modin/core/storage_formats/base/query_compiler.py

```diff
@@ -4358,8 +4366,19 @@ def write_items(df, broadcasted_items):
             df.iloc[row_numeric_index, col_numeric_index] = broadcasted_items
             return df
 
+        if not is_scalar(item):
+            broadcasted_item, _ = broadcast_item(
+                self,
+                row_numeric_index,
+                col_numeric_index,
+                item,
+                need_columns_reindex=need_columns_reindex,
+            )
+        else:
+            broadcasted_item, _ = item, pandas.Series([np.array(item).dtype])
```

Comment:
huh, what's the point of assigning `pandas.Series()` to the `_` variable here?..

Reply:
fixed